src/pages/index.js

```diff
@@ -88,19 +89,23 @@ class Index extends React.Component {
 
     let courseButtons
     if (this.props.courses && this.props.courses.length > 0) {
-      courseButtons = this.props.courses.map(course => (
-        <Link
-          route="course"
-          params={{ id: course.id }}
-          key={course.id}
-          prefetch
-          passHref
-        >
-          <Button color="primary" tag="a" className="mr-3 mb-3" outline>
-            {course.name}
-          </Button>
-        </Link>
-      ))
+      courseButtons = this.props.courses.map(course => {
+        if (!course.isUnlisted) {
```

Comment:
This will prevent both global admins and course staff from seeing their own courses. Ideally, this filtering can be done server-side, taking into account if the current authenticated user is an admin and which courses they are staff for.